### PR TITLE
Feature/add rest interfaces

### LINKED ISSUE #87: Overworld: Backend
## Requirement
The component diagram

## Current Behavior

We do not have a backend that provides information about the areas, tasks or players.

## Expected Behavior

A backend which provides and stores informations about our worlds, dungeons, tasks and players.

## Linked documents
[Database model](https://github.com/Gamify-IT/docs/blob/main/dev-manuals/services/overworld/database-model-and-type.md)
[Class-diagram](https://github.com/Gamify-IT/docs/blob/main/dev-manuals/services/overworld/class-diagram/overworld-class-diagram.png) (just area, tasks relations)

## DoD

- [x] get request for minigame configuration

### PR DESCRIPTION
**Pull request by LeviUniAcc**
Closes https://github.com/Gamify-IT/issues/issues/87

### CODE REVIEW

**Review by marlau97 — APPROVED** [review]

**Review by Aaron2000119 — APPROVED** [review]

File: src/main/java/com/example/overworldbackend/controller/OverworldController.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import java.util.Optional;
+
+
+@RestController
+public class OverworldController {
+
+    @Autowired
+    OverworldRepository overworldRepository;
+
+    @PostMapping("/save-minigame-config")
```
Comment (madebyTimo) [inline src/main/java/com/example/overworldbackend/controller/OverworldController.java]: please use better mappings. Maybe something like
```suggestion
    @PostMapping("/config/minigame")
```

File: src/main/java/com/example/overworldbackend/controller/OverworldController.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    public Configuration saveConfiguration(@RequestBody Configuration configuration) {
+        return overworldRepository.save(configuration);
+    }
+
+    @PutMapping("/change-minigame-config")
```
Comment (madebyTimo) [inline src/main/java/com/example/overworldbackend/controller/OverworldController.java]:
```suggestion
    @PutMapping("/config/minigame")
```

File: src/main/java/com/example/overworldbackend/controller/OverworldController.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    public Configuration changeConfiguration(@RequestBody Configuration configuration) {
+        Optional<Configuration> configurationToChange = overworldRepository.findById(configuration.getId());
+        if(configurationToChange.isEmpty()){
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "There is no configuration with id"+ configuration.getId());
+        }
+        configurationToChange.get().setConfigurationString(configuration.getConfigurationString());
+        return overworldRepository.save(configurationToChange.get());
+    }
+
+    @GetMapping("/get-configurationString-by-staticWorldId/{staticWorldId}")
```
Comment (madebyTimo) [inline src/main/java/com/example/overworldbackend/controller/OverworldController.java:34]:
```suggestion
    @GetMapping("/world/{staticWorldId}")
```

Comment (LeviUniAcc) [reply src/main/java/com/example/overworldbackend/controller/OverworldController.java:34]: maybe later

**Pull request merged**